apps/src/templates/HintDialogContent.jsx

```diff
@@ -0,0 +1,28 @@
+var msg = require('../locale');
+var Lightbulb = require('./Lightbulb.jsx');
+var ReadOnlyBlockSpace = require('./ReadOnlyBlockSpace.jsx');
+
+var HintDialogContent = React.createClass({
+  propTypes: {
+    content: React.PropTypes.string.isRequired,
+    block: React.PropTypes.object,
+  },
+
+  render: function () {
+    var block;
+    if (this.props.block) {
+      block = (<ReadOnlyBlockSpace block={this.props.block} />);
+    }
+
+    return (<div>
```

Comment:
I do like style consistency and that is the style I've seen everyone else using, but it makes me nervous. The visual difference between 

```
return (
  <div>
    <h3>
    ...
    </h3>
  </div>
);
```

and

```
return
  <div>
    <h3>
    ...
    </h3>
  </div>
```

is relatively minor, and the latter is actually valid JS that will compile without errors, only to throw a `Uncaught Invariant Violation: DialogButtons.render(): A valid ReactComponent must be returned. You may have returned undefined, an array or some other invalid object.` error at runtime.